src/main/java/frc/trigon/robot/subsystems/arm/simulationarm/SimulationArmIO.java

```diff
@@ -0,0 +1,96 @@
+package frc.trigon.robot.subsystems.arm.simulationarm;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.ElevatorSim;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import frc.trigon.robot.constants.RobotConstants;
+import frc.trigon.robot.subsystems.arm.ArmIO;
+import frc.trigon.robot.subsystems.arm.ArmInputsAutoLogged;
+
+public class SimulationArmIO extends ArmIO {
+    private double
+            angleVoltage = 0,
+            elevatorVoltage = 0;
+    private final SingleJointedArmSim
+            masterAngleMotor = SimulationArmIOConstants.MASTER_ANGLE_MOTOR,
+            followerAngleMotor = SimulationArmIOConstants.FOLLOWER_ANGLE_MOTOR;
+    private final ElevatorSim
+            masterElevatorMotor = SimulationArmIOConstants.MASTER_ELEVATOR_MOTOR,
+            followerElevatorMotor = SimulationArmIOConstants.FOLLOWER_ELEVATOR_MOTOR;
+
+    @Override
+    protected void updateInputs(ArmInputsAutoLogged inputs) {
+        masterAngleMotor.update(RobotConstants.PERIODIC_TIME_SECONDS);
+        followerAngleMotor.update(RobotConstants.PERIODIC_TIME_SECONDS);
+        masterElevatorMotor.update(RobotConstants.PERIODIC_TIME_SECONDS);
+        followerElevatorMotor.update(RobotConstants.PERIODIC_TIME_SECONDS);
+
+        inputs.angleMotorVoltage = angleVoltage;
+        inputs.angleMotorCurrent = masterAngleMotor.getCurrentDrawAmps();
+        inputs.angleVelocityDegreesPerSecond = getAngleVelocityDegreesPerSecond();
+
+        inputs.elevatorMotorVoltage = elevatorVoltage;
+        inputs.elevatorMotorCurrent = masterElevatorMotor.getCurrentDrawAmps();
+        inputs.elevatorPositionRevolution = masterElevatorMotor.getPositionMeters();
+        inputs.elevatorVelocityRevolutionsPerSecond = masterElevatorMotor.getVelocityMetersPerSecond();
```

Comment:
You're putting meters position for revolutions

Reply:
I couldn't find a conversion from meters

Reply:
Save a constant of meters per revolution